src/components/AdminPane/Manage/Widgets/ProjectManagersWidget/ProjectManagersWidget.js

```diff
@@ -42,10 +42,14 @@ export default class ProjectManagersWidget extends Component {
     choosingTeam: false,
   }
 
-  componentDidMount() {
-    this.props.fetchProjectManagers(this.props.project.id).then(() =>
-      this.setState({loadingManagers: false})
-    )
+  async componentDidMount() {
+    try {
+      await this.props.fetchProjectManagers(this.props.project.id)
+      this.setState({ loadingManagers: false })
+    } catch (error) {
+      console.error("Error fetching project managers:", error)
+      this.setState({ loadingManagers: false })
+    }
```

Comment:
this logicially doesn't solve anything and using async in a lifecycle method is an antipattern

Reply:
Im not even seeing the error anymore with the previous code, so ill just remove this pr for now and come up with a better solution if the error happens again.